pkg/cluster/install.go

```diff
@@ -122,22 +123,22 @@ func (i *Installer) Install() error {
 
 // PreFlightCheck checks whether install can run or not.
 func (i *Installer) PreFlightCheck() error {
-	fmt.Print("Checking if flux namespace exists...")
+	log.Actionf("checking that flux namespace exists")
 	args := []string{"get", "namespace", i.FluxNamespace, "--output", "name"}
 	if output, err := i.Runner.Run(kubectlCmd, args...); err != nil {
-		fmt.Println("\nOutput from kubectl command: ", string(output))
+		log.Waitingf("output from kubectl command: %v", string(output))
 		if i.IgnorePreflightErrors {
-			fmt.Println("WARNING: failed to get flux namespace. Flux is required for profiles to work.")
+			log.Warningf("WARNING: failed to get flux namespace. Flux is required for profiles to work.")
 		} else {
-			return fmt.Errorf("failed to get flux namespace: %w\nTo ignore this error, please see the  --ignore-preflight-checks flag.", err)
+			return fmt.Errorf("failed to get flux namespace: %w\nTo ignore this error, please see the  --ignore-preflight-checks flag", err)
 		}
 	}
-	fmt.Println("done.")
-	fmt.Print("Checking for flux CRDs...")
+	log.Successf("found flux namespace '%v'", i.FluxNamespace)
+	log.Actionf("checking for flux CRDs")
 	output, err := i.Runner.Run(kubectlCmd, "get", "crds", "--output", "jsonpath='{.items[*].spec.names.singular}'")
 	if err != nil {
 		if i.IgnorePreflightErrors {
-			fmt.Println("WARNING: failed to list all installed crds. Flux is required for profiles to work.")
+			log.Warningf("WARNING: failed to list all installed crds. Flux is required for profiles to work.")
```

Comment:
Do we need the "WARNING" text if its got the warning symbol :thinking: ? Maybe it makes it more clear, or perhaps we can move the "WARNING" text into the func itself. Wdyt?

Reply:
Good idea! Consistency is key, right??!!